src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -159,6 +159,9 @@ export default class Selection extends React.Component<Props> {
                 item_disabled_condition: {
                     value: itemDisabledCondition,
                 } = {},
+                allow_deselect_for_disabled_items: {
+                    value: allowDeselectForDisabledItems,
```

Comment:
I think the default value here should be `true`, because by default the item should be able to be removed. It is strange that an item that is actually not even allowed to be there cannot be even removed :see_no_evil: